evm_arithmetization/src/cpu/kernel/tests/init_exc_stop.rs

```diff
@@ -89,19 +91,47 @@ fn test_init_exc_stop() {
         "Incorrect registers for dummy run."
     );
 
-    let main_offset = KERNEL.global_labels["main"];
-    let mut interpreter: Interpreter<F> =
-        Interpreter::new_dummy_with_generation_inputs(initial_offset, vec![], &inputs);
+    let exc_stop_offset = KERNEL.global_labels["exc_stop"];
+
+    let pc_u256 = U256::from(interpreter.get_registers().program_counter);
+    let exit_info = pc_u256 + (U256::one() << 32);
+    interpreter.push(exit_info).unwrap();
+    interpreter.get_mut_registers().program_counter = exc_stop_offset;
     interpreter.halt_offsets = vec![KERNEL.global_labels["halt_final"]];
     interpreter.set_is_kernel(true);
     interpreter.clock = 0;
+
+    // Set the program counter and `is_kernel` at the end of the execution. They
+    // have offsets 6 and 7 respectively in segment `RegistersStates`.
```

Comment:
Out of safety with these floating indices, we should probably define a new enum, similarly to how `ContextMetadata`, `GlobalMetadata`, `TxnFields` are being defined, to ensure we never mess up an index.